data/human/weapons.txt

```diff
@@ -1005,6 +1005,37 @@ outfit "Javelin Mini Pod"
 	description "	The weapon's high rate of fire also makes it useful for saturating well-protected ships' anti-missile defenses so that more dangerous projectiles fired by allied ships can sneak through."
 	description "	Mini pods enable fighters and other light craft to carry small quantities of Javelins into battle while retaining enough space for other weapons. Large ships are better off with full-size pods."
 
+outfit "Javelin Turret"
+	category "Secondary Weapons"
+	cost 110000
+	thumbnail "outfit/javelin turret"
+	"mass" 15
+	"outfit space" -31
+	"weapon capacity" -31
+	"turret mounts" -1
+	"javelin capacity" 400
+	weapon
+		sprite "projectile/javelin"
+		"hardpoint sprite" "hardpoint/javelin turret"
+		"hardpoint offset" 9.
+		sound "javelin"
+		ammo "Javelin"
+		icon "icon/javelin"
+		"hit effect" "tiny explosion"
+		"inaccuracy" 4
+		"turret turn" 2
+		"velocity" 10
+		"lifetime" 60
+		"reload" 7
+		"firing energy" .2
+		"firing heat" 12
+		"shield damage" 48
+		"hull damage" 26
+		"hit force" 50
+		"missile strength" 3
+	description "Mounting not one but two Javelin missile pods, the Javelin turret rains down an endless stream of unguided missiles that deal noticeably more damage than other turreted weapons of the same size, at the expense of using a prodigious amount of ammo due to its high fire rate. For that reason, these turrets are commonly paired with Javelin storage racks, as well as with other weapons that don't require the use of ammunition to fall back on once all of the rockets are expended."
+	description "	As with the pod version, the Javelin turret excels at overwhelming anti-missile systems - especially due to its high fire rate, which is just over double that of the Javelin pod - so that other ordnance from allied ships is able to reach the target."
```

Comment:
```suggestion
	description "Mounting not one but two Javelin Pods, the Javelin Turret rains down an endless stream of unguided missiles that deal noticeably more damage than other turreted weapons of the same size, at the expense of using a prodigious amount of ammo due to its high fire rate. For that reason, these turrets are commonly paired with Javelin Storage Crates, as well as with other weapons that don't require the use of ammunition to fall back on once all of the rockets are expended."
	description "	With just over twice the fire rate of a Javelin Pod, the Javelin Turret excels at overwhelming anti-missile systems so that other ordnance from allied ships is able to reach the target."
```
See #8357 for capitalisations.

The second paragraph read very weirdly to me, and I've juggled it around a bit, but I'd like to get a second opinion on this one.

Reply:
You're right. This is better and more readable than the text with the dashes.